concepts/objects/about.md

```diff
@@ -0,0 +1,260 @@
+# About
+
+To use terminology from other languages, a JSON object is a "hash" or "map" or "dictionary".
+
+JSON defines an object as:
+
+> An object is an unordered set of **name**/**value** pairs.
+> An object begins with `{`left brace and ends with `}`right brace.
+> Each _name_ is followed by `:`colon and the _name/value_ pairs are separated by `,`comma.
+
+Further, the _name_ must be a string.
+Another word for _name_ is _key_.
+
+The _value_ can be of any JSON type.
+Different keys in the object can have values of different type, like this example.
```

Comment:
```suggestion
Different _values_ in the same object can be different type, like this example.
```

Using "keys" and "values" both in the sentence makes it a bit confusing about which you're talking about.